stubs/pycocotools/METADATA.toml

```diff
@@ -1,2 +1,5 @@
 version = "2.0.*"
 upstream_repository = "https://github.com/ppwwyyxx/cocoapi"
+requires = ["numpy>=2.0.0rc1"]
```

Comment:
```suggestion
requires = ["numpy>=2.0.0rc1; python_version>='3.9'"]
```

Reply:
Thanks! This worked.

For future me who will read this comment: the semicolon is important here. In the whole typeshed used only here and in a seaborn stub.